### PR TITLE
Fixes issue #1642: Drop-down menu does not disappear

### LINKED ISSUE #1642: Menu does not disappear after launching „Start Task“ dialog
After clicking actions → start task, the menu stays open and atop the dialog which is clearly a usability bug

Screenshot from 2020-06-20 16-43-34

### PR DESCRIPTION
**Pull request by mheyen**
The drop-down-menu in the adminUI now disappears after launching „Start Task“ dialog. 
This also works for:
- The language menu, 
- if you select ? → Keyboard shortcuts, 
- and the other links in that menu.

Added control for click event in modules/admin-ui/src/main/webapp/scripts/shared/directives/oldDropdownDirective.js 

This fixes #1642

### CODE REVIEW

File: modules/admin-ui/src/main/webapp/scripts/shared/directives/oldDropdownDirective.js [hunk 1]
```diff
@@ -28,9 +28,12 @@ angular.module('adminNg.directives')
     },
     link: function ($scope, element) {
       element.on('click', function (event) {
+        var is_active = angular.element(this).hasClass('active');
         angular.element('[old-admin-ng-dropdown]').removeClass('active');
```
Comment (lkiesow) [inline modules/admin-ui/src/main/webapp/scripts/shared/directives/oldDropdownDirective.js]: Instead of re-activating, it might be simpler to spare `this` from deactivation instead?
```suggestion
        angular.element('[old-admin-ng-dropdown]').not(this).removeClass('active');
```

**Review by lkiesow — CHANGES_REQUESTED** [review]
Seems to work. It does break the tests though.
This might be an issue with the tests or a sign that something is wrong in some edge cases.
We need to make sure the tests run through.

I've also suggested an alternative which might be a bit easier though I don't have a strong opinion on this.

**Review by lkiesow — APPROVED** [review]
